### DISCUSSION TITLE
Is ANTD good for e-commerce website? 👀

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#42146] [Q&A]
Hi all, I'd like to use antd for my ecommerce frontend, is it a good choice? I mean, on dektop it's good but on mobile seems like it's not so **"touchable"**, do you know any websites using antd for their frontend? Thanks 🙏
Also, how I can use my custom icons library?

**Comment by zombieJ** [comment]
For mobile support, we have the plan to enhance it. But it's long way plan since we still focus on the desktop first. Component provides `xxxIcon` prop that you can use it. It's not hard deps.

**Reply by vitalijalbu** [reply]
Yes I understand... can it be done anyway by adding some aria props to buttons?

**Reply by yoyo837** [reply]
Please help to improve that.

**Reply by vitalijalbu** [reply]
Sure, I will, so your advice is a **yes**, can use it and improve continously?

**Reply by zombieJ** [reply]
Yes. We are continually to improve  a11y ability

**Comment by yoyo837** [comment]
Check out https://github.com/ant-design/ant-design-mobile.
